analyzer/codechecker_analyzer/cmd/analyzers.py

```diff
@@ -119,8 +119,13 @@ def main(args):
         if args.dump_config == 'clang-tidy':
             subprocess.call([binary, '-dump-config', '-checks=*'])
         elif args.dump_config == 'clangsa':
-            # TODO: Not supported by ClangSA yet!
-            LOG.warning("'--dump-config clangsa' is not supported yet.")
+            ret = subprocess.call([binary, '-cc1',
+                                   '-analyzer-checker-option-help'],
```

Comment:
Mind that this wont list alpha options, you'd need to add `-analyzer-checker-option-help-alpha`. Following the theme of alpha checkers, they are considered incomplete (even if its attached to an otherwise non-alpha checker), but since we allow enabling alpha checkers, we could consider listing them as well.